tests/test_inference.py

```diff
@@ -55,12 +55,11 @@ def test_predict() -> None:
     assert all(note_pitch_max)
     assert isinstance(note_events, list)
 
-    expected_model_output = np.load("tests/resources/vocadito_10/model_output.npz")
+    expected_model_output = np.load("tests/resources/vocadito_10/model_output.npz", allow_pickle=True)["arr_0"].item()
```

Comment:
could you explain what's happening here a bit? I guessed from above that you were forcing a lowered version of tensorflow in order to avoid any issues with our current serialization, but this line makes it seem that you might be adapting to the new serialization format? If that makes sense.

Reply:
I had to remake all the test data and this is what made it work. The new format is in the keras3.0 / model loading layer. Not in numpy. There may have been some changes to the numpy format along the way.